code/weapon/weapons.cpp

```diff
@@ -2186,6 +2186,45 @@ int parse_weapon(int subtype, bool replace, const char *filename)
 		}
 	}
 
+	int num_spawn_intervals_defined = 0;
+
+	while (optional_string("$Spawn Interval:"))
+	{
+		stuff_float(&dum_float);
+
+		if (num_spawn_intervals_defined < MAX_SPAWN_TYPES_PER_WEAPON)
+		{
+			wip->spawn_info[num_spawn_intervals_defined].spawn_interval = dum_float;
+			if (dum_float > 0)
+				wip->maximum_children_spawned += (int)(wip->spawn_info[num_spawn_intervals_defined].spawn_count / wip->lifetime);
```

Comment:
You should probably put a safety cap on `dum_float` to avoid really small values.